cmd/cluster-olm-operator/main.go

```diff
@@ -105,13 +106,28 @@ func runOperator(ctx context.Context, cc *controllercmd.ControllerContext) error
 		deploymentControllerList = append(deploymentControllerList, controller)
 	}
 
+	operatorImageVersion := status.VersionForOperatorFromEnv()
```

Comment:
Re: my other comment about terminology: this would still be "operator" since `cluster-olm-operator` _is_ an operator